### PR TITLE
Sn /feature readinglist display

### LINKED ISSUE #7: Nav Component
At the top of the screen, there should be a `Nav` component that redirects the user to different parts of the page
- [ ] Header text, that takes the users home `/`
- [ ] Reading List button that takes users to `/reading-list`

### LINKED ISSUE #22: Redux implementation - add 'toReadList' to store
- [ ] Add action and reducer to connect readingList array to store

### PR DESCRIPTION
**Pull request by NakiNorton**
#### What's this PR do?
- Creates new components that renders the users Reading List
- Enables user navigation to the Reading List 'page' by connecting the file path with the NavLink 
- Styles the Reading List and adds an animation to display if there are no books to render
- Moves the ReadingList from this.state to be in the Redux 'store', adding an addFavorite action and ReadingList reducer to enable this functionality

Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

#### Where should the reviewer start?
- Start at Books.js where ReadingList is rendered. Then review ReadingList and ReadingListBook components. Look at the actions and reducer files to see Redux changes

#### Why is this change required? What problem does it solve?
- Enables user to view their ReadingList
- readingList array is more easily accessible with Redux store

#### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
- Getting readingList back from store was challenging. As a team we were able to figure out the correct return statement to get the data we needed

#### Any background context you want to provide?
- n/a

#### What are the relevant tickets?
- closes #7 
- closes #22

#### Screenshots (if appropriate)
- n/a

#### Questions:
- n/a

### CODE REVIEW

File: src/Components/ReadingList/ReadingList.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  opacity:0;
+}
+
+.eyebrow:nth-child(1){
+  transform:rotate(-20deg);
+}
+
+.eyebrow:nth-child(2){
+  transform:rotate(20deg);
+}
```
Comment (NicoleLatifi) [inline src/Components/ReadingList/ReadingList.css:100]: LOVE this error handling message. Great user experience.

File: src/Components/ReadingListBook/ReadingListBook.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  border-radius: 0.3em;
+  box-shadow: 2px 2px 15px  #C09081;
+}
+
+.remove-book:hover {
+  background-color: teal;
+  color: white;
+  border: solid 1px  white;
+  box-shadow: 2px 2px 15px  whitesmoke;
+}
```
Comment (NicoleLatifi) [inline src/Components/ReadingListBook/ReadingListBook.css:35]: I really like this hover style. Should we add this to the home page?

**Review by NicoleLatifi — APPROVED** [review]
The functionality for adding a book to the reading list is looking great!

**Review by amykarnaze — APPROVED** [review]
Praise: nice job moving readingList to store! UI looks good and LOVE the error handling illustration.

File: src/Components/ReadingList/ReadingList.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  width:45px;
+  top:62px;
+}
+
+.tear1{
+  left:29px;
+  border-bottom-left-radius:50%;
+}
+
+.tear2{
```
Comment (kip-west) [inline src/Components/ReadingList/ReadingList.css:69]: I'm so excited about this animation!! Such a cute/awesome addition.

File: src/Components/ReadingListBook/ReadingListBook.css (hunk 2 above)
Comment (kip-west) [reply src/Components/ReadingListBook/ReadingListBook.css:35]: I think so!! Would we have to change the whole card to be clickable then?

Comment (amykarnaze) [reply src/Components/ReadingListBook/ReadingListBook.css:35]: Definitely agree!

File: src/reducers/ReadingList.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export const readingList = (state = [], action) => {
+  switch (action.type) {
+    case 'SET_READING_LIST':
+      return ( [...state, action.readingList] )
+    default:
```
Comment (kip-west) [inline src/reducers/ReadingList.js:5]: Great work getting this info into `store`!

nitpick: we'll need to rename `readingList` to `book` in the future.

**Review by kip-west — APPROVED** [review]

**Pull request merged**